src/site/content/en/notifications/use-push-notifications-to-engage-users/index.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Use push notifications to engage and re-engage users
+authors:
+  - katejeffreys
+description: |
+  Use push notifications to engage users with targeted, timely updates.
+date: 2019-10-21
+codelabs:
+  - codelab-notifications-get-started
+  - codelab-notifications-service-worker
+---
+
+## Why use push notifications?
+
+Notifications present small chunks of information to a user. Web apps can use notifications to tell users about important, time-sensitive events, or actions the user needs to take. 
+
+The look-and-feel of notifications varies between platforms. For example: 
+
+<figure class="w-figure w-figure--center">
+  <img class="w-screenshot w-screenshot--filled" src="./predicaments-android.png" alt="">
+  <figcaption class="w-figcaption">A notification on an Android device.</figcaption>
+</figure>
+
+<figure class="w-figure w-figure--center">
+  <img class="w-screenshot w-screenshot--filled" src="./predicaments-macbook.png" alt="">
+  <figcaption class="w-figcaption">A notification on a MacBook.</figcaption>
+</figure>
+
+Traditionally, web browsers had to initiate the exchange of information between server and client by making a request. [Web push technology](https://developer.mozilla.org/en-US/docs/Web/API/Push_API), on the other hand, lets you configure your server to send notifications when it makes sense for your app. A push service creates unique URLs for each subscribed service worker. Sending messages to a service worker's URL raises events on that service worker, prompting it to display a notification.      
```

Comment:
This is the first mention of 'push service'. I think it would be good to explain this first — something like this:

> Each browser has a push service back end to forward messages to the correct user. When a user agrees to get notifications from your site, you can use JavaScript to get a unique subscription ID for them from their browser's push service.
> 
> Using that ID, you send messages from your application server to the push service, which then forwards them to the user. This process uses the Push API.
> 
> On the client side, the operating system notifies the browser when a push message is received. Service workers can listen out for push events — even when the browser is closed — and display a notification using the Notification API.